src/status_im/config.cljs

```diff
@@ -48,7 +48,7 @@
 
 (def opensea-api-key OPENSEA_API_KEY)
 (def status-proxy-enabled? true)
-(def status-proxy-stage-name (get-config :STATUS_PROXY_STAGE_NAME "test"))
+(def status-proxy-stage-name (get-config :STATUS_PROXY_STAGE_NAME "prod"))
```

Comment:
intentional??

Reply:
yes, but not meant to be committed, but qa will want to test against prod (same as release)